### PR TITLE
Fix 'activate changes' in helpers module

### PR DESCRIPTION
**Pull request by MatteoStifano**
Make sure the cmk-changes-activation process is complete and response status '204' is returned.

### CODE REVIEW

File: tests/cypress/e2e/helpers.ts [hunk 1]
```diff
@@ -55,14 +55,45 @@ export function deleteCmkHost(hostName: string) {
...
+      Cypress.env('cypressToCheckmkUrl') +
+      '/check_mk/api/1.0/objects/activation_run/' +
+      activation_id +
+      '/actions/wait-for-completion/invoke',
+    followRedirect: false,
+    headers: { accept: 'application/json' },
+    auth: {
+      bearer: `${Cypress.env('cmkUsername')} ${Cypress.env('cmkPassword')}`,
+    },
+    body: {
```
Comment (BenediktSeidl) [inline tests/cypress/e2e/helpers.ts:72]: body should not be needed here.

File: tests/cypress/e2e/helpers.ts [hunk 2]
```diff
@@ -55,14 +55,45 @@ export function deleteCmkHost(hostName: string) {
...
+    },
+  }).then((response) => {
+    if (response.status === 204) return;
+
+    waitForActivation(activation_id);
+  });
+}
+
 export function activateCmkChanges(siteName: string) {
+  Cypress.on('uncaught:exception', (err) => {
```
Comment (BenediktSeidl) [inline tests/cypress/e2e/helpers.ts:83]: are you sure this fixes `Could not find data source` of https://github.com/tribe29/grafana-checkmk-datasource/actions/runs/3722750224/jobs/6313829415 ? 
`Could not find data source` is a grafana error. do you know when/why this error occurs? could there be another solution? it feels strange that we catch a grafana error here in the checkmk code.
my guess would be, that this is an error from the previous check (configuring the data source). can you investigate here?
this looks a bit too general for me. if possible we should find the root cause and fix it there.

File: tests/cypress/e2e/helpers.ts (hunk 1 above)
Comment (MatteoStifano) [reply tests/cypress/e2e/helpers.ts:75]: Removed.

**Comment by BenediktSeidl** [comment]
i watched the cypress video, and it seems to me, that grafana can not reach checkmk:

local:



cypress:

File: tests/cypress/e2e/helpers.ts (hunk 2 above)
Comment (MatteoStifano) [reply tests/cypress/e2e/helpers.ts]: Discussed f2f.

**Review by BenediktSeidl — APPROVED** [review]

**Pull request merged**